### PR TITLE
Fix testnet launch profile (TumbleBit run profile was accidentally co…

### PR DESCRIPTION
**Pull request by nopara73**
…mmitted by someone before)

### CODE REVIEW

File: Breeze/src/Breeze.Daemon/Properties/launchSettings.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     },
     "Breeze.Daemon TestNet": {
       "commandName": "Project",
-      "commandLineArgs": "-testnet -tumbler-uri=http://localhost:5050 light -debug=all -loglevel=debug"
+      "commandLineArgs": "light -testnet"
```
Comment (bokobza) [inline Breeze/src/Breeze.Daemon/Properties/launchSettings.json]: Why remove the debug arguments?

Comment (nopara73) [reply Breeze/src/Breeze.Daemon/Properties/launchSettings.json]: I've never used them, nor seen anyone using them and they're not documented. What do they do? Should I keep them there?

Comment (bokobza) [reply Breeze/src/Breeze.Daemon/Properties/launchSettings.json]: Yes please keep them there, they're use to set the level of debugging we wish to have while running this project.
You can remove the tumbler thingy though, that's a thing from the past.

**Comment by bokobza** [comment]
all good mate

**Pull request merged**